packages/rendermime/src/renderers.ts

```diff
@@ -398,6 +399,14 @@ export function renderMarkdown(
       if (shouldTypeset && latexTypesetter) {
         latexTypesetter.typeset(host);
       }
+    })
+    .then(() => {
```

Comment:
I think this `.then` clause can be consolidated with the one above, since it resolves more or less immediately (the typesetting is asynchronous, but we don't wait on it).

Reply:
OK. If we don't wait for LaTeX typesetting, we can join the two clauses.